### ISSUE TITLE
confd has no explicit log for failed bind to socket

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#350]
Originally reported of Google Code with ID 294.

```
Currently, if confd can't bind to the luxi socket, this is what happens:

iustin@rrr:~/work/ganeti$ ./htools/hconfd -d --no-user -f
2012-10-31 16:43:27 CET: ganeti-confd pid=6948 NOTICE starting
2012-10-31 16:43:27 CET: ganeti-confd pid=6948 INFO Loaded new config, serial 22110
2012-10-31 16:43:27 CET: ganeti-confd pid=6948 ERROR Error while opening luxi socket: bind: does not exist (No such file or directory)
hconfd: bind: does not exist (No such file or directory)
2012-10-31 16:43:33 CET: ganeti-confd pid=6948 INFO Moving to inotify mode

But no other error - as the exception is not caught.

We should at least log this better, but I'd prefer if confd refuses to start if binding to the socket fails, as this is not a normal state - and failing to start is preferable, for easier diagnosis.
```
Originally added on 2012-10-31 15:46:24 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2013-04-08 20:16:38 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
I've actually fixed this via the redoing of startup protocol. Now confd doesn't run if it can't bind anymore.
```
Originally added on 2013-04-09 07:47:22 +0000 UTC.

Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Marking 2.7 bugs fixed, as 2.7.0~rc1 was released, and should be ready to be tested in production environments.
```
Originally added on 2013-05-09 05:42:48 +0000 UTC.

Changed State: Released
